resources/keywords/navigation.resource

```diff
@@ -0,0 +1,26 @@
+*** Settings ***
+Library    String
+Resource    ../OR/login_obj.resource
+Resource    ../Keywords/login.resource
```

Comment:
Please change to _objectrepo_ and _keywords_ or can we just have ../login.resource